crates/cast/bin/cmd/send.rs

```diff
@@ -171,14 +194,57 @@ impl SendTxArgs {
 
             tx::validate_from_address(eth.wallet.from, from)?;
 
-            let (tx, _) = builder.build(&signer).await?;
-
-            let wallet = EthereumWallet::from(signer);
-            let provider = ProviderBuilder::<_, _, AnyNetwork>::default()
-                .wallet(wallet)
-                .on_provider(&provider);
-
-            cast_send(provider, tx, cast_async, confirmations, timeout, to_json).await
+            if zksync_params.zksync {
+                // ZkSync transaction
+                let paymaster_address = zksync_params
+                    .paymaster_address
+                    .as_ref()
+                    .map(|s| Address::from_str(s))
+                    .transpose()?
+                    .map(|addr| addr.to_h160())
+                    .unwrap_or_default();
+                let paymaster_input = zksync_params
+                    .paymaster_input
+                    .as_ref()
+                    .map(|s| Bytes::from_str(s))
+                    .transpose()?
+                    .map(|bytes| bytes.to_vec())
+                    .unwrap_or_default();
```

Comment:
I mean, this works [afaik](https://github.com/popzxc/alloy-zksync/blob/63359d567025e41a181ed6a413ea6d9c52faf1ce/src/network/unsigned_tx/eip712/signing.rs#L10-L21), but I was thinking some more along the lines of `Option::and_then`: `paymaster_address.and_then(|addr| paymaster_input.map(|input| (addr, input)).map(|(addr, input)| PaymasterParams { ... })` and passing that in...
So we don't pass any "default" value if not present, and we can leave the encoding to implementation details